NitroxModel/Discovery/InstallationFinders/EnvironmentFinder.cs

```diff
@@ -0,0 +1,37 @@
+using NitroxModel.Discovery.InstallationFinders.Core;
+using NitroxModel.Discovery.Models;
+using System;
+using static NitroxModel.Discovery.InstallationFinders.Core.GameFinderResult;
+
+namespace NitroxModel.Discovery.InstallationFinders;
+
+/// <summary>
+/// Trying to find the path in environment variables by the key {GAMEINFO FULLNAME}_INSTALLATION_PATH that contains the installation directory.
+/// <list>
+///     <item>SUBNAUTICA_INSTALLATION_PATH</item>
+///     <item>SUBNAUTICAZERO_INSTALLATION_PATH</item>
+/// </list>
+/// </summary>
+public sealed class EnvironmentFinder : IGameFinder
+{
+    public GameFinderResult FindGame(GameInfo gameInfo)
+    {
+        string path = Environment.GetEnvironmentVariable($"{gameInfo.FullName.ToUpper()}_INSTALLATION_PATH");
+        if (string.IsNullOrEmpty(path))
+        {
+            return Error($"Configured game path with environment variable {gameInfo.FullName.ToUpper()}_INSTALLATION_PATH was found empty");
+        }
```

Comment:
`gameInfo.FullName` returns "Subnautica: Below Zero", not "SubnauticaZero". Should probably be `gameInfo.Name` here.